src/js/__depricated__/jwplayer.embed.flash.js

```diff
@@ -247,7 +247,7 @@
         if (utils.isRtmp(file, type)) { return true; }
         if (type === 'hls') { return true; }
 
-        var mappedType = utils.extensionmap[type ? type : utils.extension(file)];
+        var mappedType = utils.extensionmap.getMappedType(type ? type : utils.extension(file));
```

Comment:
ternary conditionals like this should just use "or" operator. It's a dep script so ignoring.